central/debug/service/service.go

```diff
@@ -598,6 +620,30 @@ func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseW
 
 	fetchAndAddJSONToZip(debugDumpCtx, zipWriter, "system-configuration.json", s.getConfig)
 
+	// Fetch alerts and append to the zip
+	if opts.withAlerts {
+		fetchAndAddJSONToZip(debugDumpCtx, zipWriter, "alerts-before.json", s.fetchAlerts)
+		// clusterSet := set.NewFrozenStringSet(opts.clusters...)
+		for _, conn := range s.sensorConnMgr.GetActiveConnections() {
+			err := conn.InjectMessage(debugDumpCtx, &central.MsgToSensor{
+				Msg: &central.MsgToSensor_ReprocessDeployments{
+					ReprocessDeployments: &central.ReprocessDeployments{},
+				},
+			})
+			if err != nil {
+				log.Warnf("Failed to request reprocess deployment for cluster: %s", conn.ClusterID())
+				continue
+			}
+		}
+
+		log.Info("Requested reprocess clusters. Waiting for a minute before finish writing the diagnostic bundle")
```

Comment:
Why 1 minute? Is there a guarantee it will be enough?

Reply:
This is pretty arbitrary. The current approach was to set to a time where we're are confident that most alerts would have arrived after reassessing. But we don't have any way to guarantee that all _possible_ alerts were already received. 

We thought also about making this duration configurable. E.g. with a flag `--reprocess-delay 30s`?